torchrl/objectives/ddpg.py

```diff
@@ -357,3 +346,23 @@ def make_value_estimator(self, value_type: ValueEstimators = None, **hyperparams
             "done": self.tensor_keys.done,
         }
         self._value_estimator.set_keys(**tensor_keys)
+
+    @property
+    @cache_values
+    def __target_params(self):
+        target_params = TensorDict(
+            {
+                "module": {
+                    "0": self.target_actor_network_params,
+                    "1": self.target_value_network_params,
+                }
+            },
+            batch_size=self.target_actor_network_params.batch_size,
+            device=self.target_actor_network_params.device,
+        )
+        return target_params
```

Comment:
Same here there is not much benefit in chaching since all components are cached right?

Reply:
same as before: TD construction is a bottleneck